models/topics-model.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+
+const allTopics = (sort_by = "slug") => {
+  const topicObjects = ["slug", "description"];
+  const queryValues = [];
```

Comment:
We don't need to add any queries to this yet (spoiler, we'll be doing this in ticket 11!) so you can remove this bit. We're only interested in selecting all from topics for this ticket 🙂